docs/Reference/Alternative-EC-Curves.md

```diff
@@ -0,0 +1,31 @@
+description: Using alternative Elliptic Curves in Besu
+<!--- END of page meta data -->
+
+# Using alternative Elliptic Curves in Besu
+
+!!! caution
+    Alternative Elliptic Curves is an experimental feature.
+
+By default, Besu uses the Ethereum standard secp256k1 elliptic curve (EC).
+However, when running nodes in a private network, it is possible to configure an alternative elliptic curve.
+
+The configuration for what elliptic curve Besu will use is done in the network configuration section of genesis file, using the [`ecCurve`](Config-Items.md#Configuration_Items) key:
+
+```bash
+{
+  "genesis": {
+    "config": {
+      "ecCurve": "secp256k1",
+    [...]
+  },
+  [...]
+}
+```
+
+!!! attention
+    All nodes in the network **MUST** use the same elliptic curve.
```

Comment:
Could you say more about the consequences? Will it corrupt something or just break the consensus? What if it happens? does it break something and can we fix if we have a node joining with the wrong EC scheme?